cypress/components/batch-selection/batch-selection.cy.tsx

```diff
@@ -12,13 +10,26 @@ import {
   batchSelectionButtonsByPosition,
 } from "../../locators/batch-selection/index";
 
+type BATCH_SELECTION_COLOR_SCHEME_TYPE =
```

Comment:
You could derive the type from the array if you wanted

```suggestion
type BatchSelectionColorSchemeType = typeof BATCH_SELECTION_COLOR[number]

```